### PR TITLE
Armenian language support

### PR DESCRIPTION
**Pull request by movses**
Translation for the Armenian language.

### CODE REVIEW

File: src/i18n/languages.json [hunk 1]
```diff
@@ -4,6 +4,11 @@
     "name": "English (US)",
     "translated_name": "English (US)"
   },
+  {
+    "code": "am",
```
Comment (wojtekmaj) [inline src/i18n/languages.json:8]: React uses "hy" as code for Armenian, am I missing something?

File: src/i18n/json/am-AM.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "Mounting": "Մոնտաժում",
+  "Updating": "Թարմացում",
+  "Unmounting": "Ապամոնտաժում",
+  "“{name} phase”": "“Փուլ {name}”",
+  "Pure and has no side effects. May be paused, aborted or restarted by React.": "Մաքուր է և չունի կողմնակի ազդեցություններ: Կարող է կանգնեցվել, վիժեցվել կամ վերագործարկվել React-ի կողմից:",
+  "Can read the DOM.": "Կարող է կարդալ DOM-ը:",
+  "Can work with DOM, run side effects, schedule updates.": "Կարող է աշխատել DOM-ի հետ, կատարել կողմնակի էֆեկտները, պլանավորել թարմացումներ:",
+  "React updates DOM and refs": "React-ը թարմացնում է DOM-ը և ref-երը",
+  "Read docs for {name} (opens in a new tab)": "Կարդալ փաստաթղթերը {name} (բացվում է նոր թաբում)",
+  "https://reactjs.org/docs/react-component.html#{docname}": "https://reactjs.org/docs/react-component.html#{docname}",
```
Comment (wojtekmaj) [inline src/i18n/json/am-AM.json:16]:
```suggestion
  "https://reactjs.org/docs/react-component.html#{docname}": "https://hy.reactjs.org/docs/react-component.html#{docname}",
```

File: src/i18n/languages.json (hunk 1 above)
Comment (movses) [reply src/i18n/languages.json:8]: Yes, you're right.  
`AM` is country code.

File: src/i18n/json/am-AM.json (hunk 2 above)
Comment (movses) [reply src/i18n/json/am-AM.json]: Yes, sure.

**Comment by wojtekmaj** [comment]
Can you please check "Allow edits from maintainers" checkbox"?

**Comment by movses** [comment]
> Can you please check "Allow edits from maintainers" checkbox"?

It's already checked
Screen Shot 2020-07-15 at 06 35 51

**Pull request merged**